src/content/documentation/develop/request-the-right-permissions.md

```diff
@@ -102,7 +102,9 @@ Not all [permissions](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExte
 | `clipboardWrite`                                                                                                               | Input data to the clipboard                                                                                                                                                                                      |
 | `clipboardRead`                                                                                                                | Get data from the clipboard                                                                                                                                                                                      |
 | Other permissions:                                                                                                               |
-| The manifest key [`devtools_page`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/devtools_page) | Extend developer tools to access your data in open tabs                                                                                                                                                          |
+| The manifest key [`devtools_page`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/devtools_page)<sup>*<sup> | Extend developer tools to access your data in open tabs                                                                                                                                                          |
```

Comment:
```suggestion
| The manifest key [`devtools_page`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/devtools_page)<sup>*</sup> | Extend developer tools to access your data in open tabs                                                                                                                                                          |
```

Insert `/` in second `<sup>`

Reply:
Not sure if one asterisk is sufficiently obvious. Would a few extra words between the current text and the asterisk improve clarity? I.e. ending up with `The manifest key devtools_page or the "devtools" optional permission`.